Resources/public/js/views/universaldiscovery/ez-universaldiscoveryfinderexplorerlevelview.js

```diff
@@ -327,6 +333,18 @@ YUI.add('ez-universaldiscoveryfinderexplorerlevelview', function (Y) {
              * @type Number
              */
             childCount: {},
+
+            /**
+             * Boolean that tell if the view is disabled or not.
+             * The items of a disabled level view can not be explored.
+             * 
+             * @attribute disabled
+             * @default false
+             * @type Boolean
```

Comment:
related to https://github.com/ezsystems/PlatformUIBundle/pull/814/commits/858da3cc40c35c5ca1e906d0834db2b09c45b9f5#r104382949, it should configured with `writeOnce: 'initOnly';`